distributed/scheduler.py

```diff
@@ -2753,7 +2753,7 @@ def transition_waiting_processing(self, key):
             worker_msgs[worker] = [_task_to_msg(self, ts)]
 
             return recommendations, client_msgs, worker_msgs
-        except Exception as e:
+        except Exception as e:  # pragma: no cover
```

Comment:
Just curious but can the omit_lines thing be a multiline regex?

`except\sException\sas\s(\w*):\n*\s*logger.exception\(\1\)\n*\s*if\sLOG_PDB:`

should match this and we could remove all of the extra pragmas

Reply:
Just tested this locally and doesn't look like it :/. Also found some SO posts on the topic, but seems the exceptions are designed to be executed line by line.

Reply:
thanks for checking